.github/workflows/tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.16.x]
+        go-version: [1.21.x]
```

Comment:
❓ Should we add all the supported versions in the matrix perhaps? Although Go doesn't break backwards compatibility there might be features implemented in 1.16+ that'll make the code not compile in older versions.